src/main/java/org/openstreetmap/atlas/checks/validation/tag/InvalidTagsCheck.java

```diff
@@ -34,50 +39,57 @@
 
 /**
  * This flags features based on configurable filters. Each filter passed contains the
- * {@link AtlasEntity} classes to check and a {@link TaggableFilter} to test objects against. If a
- * feature is of one of the given classes and passes the associated {@link TaggableFilter} then it
- * is flagged. In addition to the taggable filters, there are two configurable boolean values,
- * "filters.resource.append.override" and "filters.resource.append". If the
- * "filters.resource.append.override" key is set to true, only the filters passed through config are
- * used to flag the atlas features. If the "filters.resource.append" is set to true, the filters
- * passed through the config are appended to the default filters that are in the "invalidTags.txt"
- * resource file.
+ * {@link AtlasEntity} classes to check and a {@link TaggableFilter} or a
+ * {@link RegexTaggableFilter} to test objects against. If a feature is of one of the given classes
+ * and passes the associated filter then it is flagged. In addition to the taggable filters, there
+ * are two configurable boolean values, "filters.resource.append.override" and
+ * "filters.resource.append". If the "filters.resource.append.override" key is set to true, only the
```

Comment:
Looks like the java docs were not updated when these params were modified in a previous PR. The current config just has "filters.resource.override". Since this check is going to be updated for https://github.com/osmlab/atlas-checks/issues/377, we can update these in the upcoming PR. @mm-ciub